doc/manuals/user/ngsiv2_implementation_notes.md

```diff
@@ -289,6 +290,22 @@ Apart from the `status` values defined for subscription in the NGSIv2 specificat
 
 [Top](#top)
 
+## Force Updatation of Entity
+NGSIv2 introduced a new URI param option(forcedUpdate) for updating the entity if the value of the attribute is same as previous value.
+If this forcedUpdate URI param option is provided along with the request of update, previous value of entity would be forcefully 
+updated with requested value irrespective of the fact that whether the value is same as previous or newly updated.
+
+Below requests can be formed with forcedUpdate:
+* `POST /v2/entities/E?options=forcedUpdate`
```

Comment:
Although it is not included in the list of the issue, I have just realized that POST /v2/entities?options=forcedUpdate should be also included. Please, add it to this list in the documenation and to the .test (if not already included).

Reply:
fixed in 6df4217